apps/paf/fields/index.js

```diff
@@ -578,7 +578,7 @@ module.exports = {
   },
   'crime-location-phone': {
     className: ['govuk-input', 'govuk-input--width-20'],
-    validate: [{ type: 'maxlength', arguments: 20 }]
+    validate: ['required', 'internationalPhoneNumber', { type: 'maxlength', arguments: 20 }]
```

Comment:
Is 20 chars enough for a full phone number? Might be worth checking on other forms what they have added here, but probably ok if that has been suggested at some point

Reply:
will 25 do ?